akka-http-core/src/test/scala/akka/http/scaladsl/GracefulTerminationSpec.scala

```diff
@@ -162,7 +162,7 @@ class GracefulTerminationSpec extends WordSpec with Matchers with BeforeAndAfter
         r2.futureValue.status should ===(StatusCodes.EnhanceYourCalm) // the injected 503 response
         r2.futureValue.entity.toStrict(1.second).futureValue.data.utf8String should ===("Chill out, man!")
 
-        val terminated: Http.HttpTerminated = Await.result(serverBinding.whenTerminated, 3.seconds)
+        val _: Http.HttpTerminated = Await.result(serverBinding.whenTerminated, 3.seconds)
```

Comment:
If we really need to type-ascribe it we can do it on the right side instead and skip the val part.

Reply:
I think we can just skip the `val` part, I was doing some "search and replace"